### DISCUSSION TITLE
Ability to disable HTTPS in 17.0.0 container

### DISCUSSION THREAD

**Discussion by benridley** [keycloak/keycloak#10186] [Q&A]
### Description

Is there a way to disable HTTPS in the 17.0.0 container? I have read through the documentation in https://www.keycloak.org/server/containers but it doesn't appear to be listed in the options.

The reason I want to do this is because in our environment we front Keycloak with a reverse proxy that handles certificates. Our preference would be to continue with this setup, unless there's a security benefit to using Keylcoak with certs directly.

Thank you!

### Motivation

For users who wish to front Keycloak with a reverse-proxy that does TLS termination.

### Details

A flag or environment variable that disables HTTPS for keycloak, documented in the container documentation.

**Comment by stianst** [comment]
Take a look at https://www.keycloak.org/server/reverseproxy (and also https://www.keycloak.org/server/enabletls). You basically need to use "KC_PROXY=edge".
